exporters/common/src/main/java/io/opentelemetry/exporter/internal/grpc/GrpcExporter.java

```diff
@@ -38,45 +41,59 @@ public final class GrpcExporter<T extends Marshaler> {
 
   private final String type;
   private final GrpcSender<T> grpcSender;
-  private final ExporterMetrics exporterMetrics;
+  private final ExporterInstrumentation exporterMetrics;
 
   public GrpcExporter(
-      String exporterName,
-      String type,
       GrpcSender<T> grpcSender,
-      Supplier<MeterProvider> meterProviderSupplier) {
-    this.type = type;
+      InternalTelemetrySchemaVersion internalTelemetrySchemaVersion,
+      ComponentId componentId,
+      ComponentId.StandardExporterType exporterType,
```

Comment:
Does this need to accept ComponentId and ComponentId.StandardExporterType? Or would ComponentId.StandardExporterType suffice because you can derive a ComponentId fromit?

Reply:
The main reason I did it this way is for easier testing: Passing in a `ComponentId` allows exact verification of the resulting attributes.
However, with the change [described here](https://github.com/open-telemetry/opentelemetry-java/pull/7265#discussion_r2097796254) this now has been changed to accepting a single `StandardComponentId`, making it a little more compact.